src/providers/krb5/krb5_renew_tgt.c

```diff
@@ -385,7 +385,7 @@ static errno_t check_ccache_files(struct renew_tgt_ctx *renew_tgt_ctx)
 {
     TALLOC_CTX *tmp_ctx;
     int ret;
-    const char *ccache_filter = "(&("SYSDB_CCACHE_FILE"=*)("SYSDB_UC"))";
+    const char *ccache_filter = SYSDB_CCACHE_FILE"=*))";
```

Comment:
Is this syntactically well formed filter? It seems like closing `))` doesn't have matching `((`.

Reply:
Of course you are right, thank you. But I do test my patches :-) so I assume ldb must be somehow permissive. Even with this bug in the code, I saw:
```
(Mon Mar 25 22:00:00 2019) [sssd[be[ipa.test]]] [check_ccache_files] (0x4000): Found [1] entries with ccache file in cache.
(Mon Mar 25 22:00:00 2019) [sssd[be[ipa.test]]] [check_ccache_files] (0x0200): Failed to check ccache file [KCM:].
(Mon Mar 25 22:00:00 2019) [sssd[be[ipa.test]]] [init_renew_tgt] (0x1000): Adding offline callback to remove renewal timer.
(Mon Mar 25 22:00:00 2019) [sssd[be[ipa.test]]] [init_renew_tgt] (0x1000): Adding renewal task to online callbacks.
```

So it did find at least one entry. Nonetheless, the issue is fixed in the latest patches.

Reply:
Ack